ckan/logic/validators.py

```diff
@@ -27,7 +27,7 @@ def owner_org_validator(key, data, errors, context):
 
     if value is missing or value is None:
         if not authz.check_config_permission('create_unowned_dataset'):
-            raise Invalid(_('A organization must be supplied'))
+            raise Invalid(_('An organization must be provided'))
```

Comment:
That is something that I though about to. But there is no way to avoid it, is there?